src/content/UserPage/UserPage.js

```diff
@@ -69,9 +69,9 @@ const UserPage = () => {
                   <div className='video-container'>
                     <iframe src='https://player.vimeo.com/video/46398191'
                       title='How it Works'
-                      frameborder='0'
+                      frameBorder='0'
                       allow='autoplay; fullscreen'
```

Comment:
Can you please remove line 73?

Reply:
Yeah, sure. Didn't touch other than the warnings though